src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -610,6 +618,22 @@ public boolean addRecipe(final IToken<?> token)
         return false;
     }
 
+    /**
+     * Updates existing requests, if they match the recipes available at this worker
+     */
+    private void updateWorkerAvailableForRecipes()
```

Comment:
We should consider making this public. We do this work in several other places today, and we should probably unify them. It's possible they are all in abstractbuildingworker, but I don't think so.

Reply:
I've checked the other places, and none of them want to update all recipes